src/ui/components/championSelect/championSelectComponent.ts

```diff
@@ -0,0 +1,79 @@
+///<reference path="../../_app.ts" />
+'use strict';
+
+import Component from "vue-class-component";
+const Vue = <vuejs.VueStatic>require("vue");
+
+import "../../css/champion-select.less";
+
+import CDNService from "../../services/cdnService.ts";
+import StaticService from "../../services/staticService.ts";
+import NetworkService from "../../services/networkService.ts";
+import SettingService from "../../services/settingService.ts";
+import ModalComponent from "../modal/modalComponent.ts";
+
+@Component({
+    template: require("./championSelectView.html")
+})
+export default class ChampionSelectComponent extends Vue {
+    private canPick: boolean;
+    private mySelectedChampion: number;
+    private remainingTime: number;
+    private players: Object[];
+    private leftTeam: number[];
+    private rightTeam: number[];
+    private timer: NodeJS.Timer;
```

Comment:
Consider clearing the timer when the component is unloaded as well. Also, this is the incorrect timer type (this runs in the browser, not node).